manifests/params.pp

```diff
@@ -22,13 +22,21 @@
   case $facts['os']['family'] {
     'Debian': {
       if $facts['os']['release']['major'] =~ /^(9|10)$/ {
+        $debian_nodejs_dev_package_name = $facts['os']['release']['major'] ? {
+          '9'     => 'nodejs-dev',
+          default => 'libnode-dev',
+        }
+        $debian_npm_package_name = $facts['os']['release']['major'] ? {
+          '9'     => false,
```

Comment:
I don't like mixing of strings and booleans, can this be done differently?

Reply:
@juniorsysadmin I did it like that to preserve the existing value of `false` for Debian 9. This is also seen in the OpenBSD and Gentoo cases in this file. I suppose we could make it `undef` instead, but that would make it inconsistent with the OpenBSD and Gentoo cases. I could change those also, but that starts to get outside the scope of my pull request.